mrp_production_byproduct_cost_share/__manifest__.py

```diff
@@ -0,0 +1,14 @@
+# Copyright 2023 ForgeFlow S.L. (https://www.forgeflow.com)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    "name": "Production By-Product Cost Share",
+    "version": "14.0.1.0.0",
+    "category": "MRP",
+    "author": "ForgeFlow, " "Odoo Community Association (OCA)",
+    "website": "https://github.com/OCA/manufacture",
+    "license": "AGPL-3",
```

Comment:
Since this module is copy of original implementation, that is under LGPL-3, not sure if it is good to switch the license, even if text of LGPL-3 is included in AGPL-3.

In any case, I am not legals expert.